packages/api-client/rollup.config.js

```diff
@@ -0,0 +1,107 @@
+import postcss from 'rollup-plugin-postcss';
+import autoprefixer from 'autoprefixer';
+import visualizer from 'rollup-plugin-visualizer';
+import { terser } from 'rollup-plugin-terser';
+import typescript from '@rollup/plugin-typescript';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
+import json from '@rollup/plugin-json';
+import { babel } from '@rollup/plugin-babel';
+import replace from '@rollup/plugin-replace';
+import { defineConfig } from 'rollup';
+import pkg from './package.json';
+
+const production = !process.env.ROLLUP_WATCH;
+
+function basePlugins() {
+    return [
+        typescript({
+            sourceMap: true,
+            declaration: true,
+            declarationDir: 'dist',
```

Comment:
The `typings` property in `package.json` still targets `dist/index.d.ts` and should be updated (now `dist/src/index.d.ts`)